smithy-aws-traits/README.md

```diff
@@ -9,7 +9,7 @@ See the Smithy specification for details on how these traits are used.
 * `aws.api#service`: Configures a Smithy service as an AWS service.
 * `aws.api#arn`: Defines the ARN template of a Smithy resource.
 * `aws.api#arnReference`: Indicates that a string shape contains an ARN.
-* `aws.api#unsignedPayload`: Configures that request payload of an
+* `aws.auth#unsignedPayload`: Configures that request payload of an
```

Comment:
This is a list of contents in the `aws.api` namespace, should the README contain new lists for `aws.auth` and `aws.protocols`?

Reply:
I'll just remove the README in favor of the spec.